src/lib/pages/schedules.svelte

```diff
@@ -49,7 +36,7 @@
       {translate('common.schedules')}
     </h1>
   </div>
-  {#if hasSchedules && !createDisabled}
+  {#if !createDisabled}
```

Comment:
Removing the `hastSchedules` check here means that this button shows up in two places on the `No Schedules Found` page. Was there a reason it was removed? 

<img width="918" alt="Screenshot 2024-01-03 at 1 03 19 PM" src="https://github.com/temporalio/ui/assets/15069288/b3e21ec1-4d7b-48a6-a6b4-33c6b6fde93e">

Reply:
We don't know the count of schedules anymore outside of the <ApiPagination> component (let:visibleItems are schedules, so only the children of ApiPagination know about the count). And that Create Schedule button is outside of it. So we could refactor a bunch of things to make it work, but it feels like it's not worth it and would add a lot of complexity/slots to ApiPagination.

Reply:
Or we can remove the button in the EmptyState

Reply:
Ah makes sense. I think that was added as part of a design request (to only show one on the page) but I can't find verification of that. We could also wrap that button in an additional `{#await fetchSchedules}` and check  `{#if schedules?.length}`.